search-commons/src/main/resources/indices/resource/settings.json

```diff
@@ -1,4 +1,6 @@
 {
+  "number_of_shards": 5,
+  "number_of_replicas": 1,
```

Comment:
This is the setting we have in production environment. Number of shards has to be greater than 1 in order to be able to reproduce the bug using test container.